i18n/english.yml

```diff
@@ -1243,6 +1243,15 @@ components:
     title: Timetable editor keyboard shortcuts
   TimezoneSelect:
     placeholder: Select timezone...
+  TripSeriesModal:
+    close: Close
+    createTripSeriesBody: Enter the start and end time for the trip series (24 hour time) and headway between trips. Click generate to create the series of trips. 
+    createTripSeriesQuestion: Create a series of trips
+    disabledTooltip: There is an issue with the input data.
```

Comment:
I don't think we need the period

Reply:
Sure thing, updated in [0d258e6](https://github.com/ibi-group/datatools-ui/pull/892/commits/0d258e683ef66e3c08bc2cb25f47eafa001a3489)